CMakeLists.txt

```diff
@@ -86,7 +86,9 @@ zephyr_compile_definitions(
 
 # @Intent: Set compiler flags to enable buffer overflow checks in libc functions
 # @config in CONFIG_NO_OPTIMIZATIONS optional : Optimizations may affect security
-toolchain_cc_security_fortify()
+if(COMMAND toolchain_cc_security_fortify)
+  toolchain_cc_security_fortify()
+endif()
```

Comment:
I believe that by-design toolchains that don't support this should declare NO-OP functions.

Correct me if I'm wrong @mped-oticon